courses/spark_for_ada_programmers/6_proof.rst

```diff
@@ -620,7 +610,7 @@ Dealing with False Alarms
 
   - :ada:`Reason` is a string literal for reviews
 
-    + Reason is repeated in analysis summary file :file:`gnatprove.out`
+    + Reason is repeated in analysis summary file :filename:`gnatprove.out`
```

Comment:
What's the difference between these two? I see :file: is used elsewhere in the sources of training material.